File: CursoPOOUber/Java/Efectivo.java
package Java;

 class Efectivo extends Payment {
     Integer id;
     float cambio;
     float monto;
     String fecha;
     String email;

     public Efectivo(float monto, String fecha, String email ,float cambio){
         super(monto, fecha, email);
         this.cambio = cambio;
         this.monto = monto;
         this.fecha = fecha;
         this.email = email;
     }

}

File: CursoPOOUber/Python/efectivo.py
from payment import Payment
class Efectivo:
    id = int
    cambio = float
    monto = float
    fecha = str
    email = str

    def __init__(self, cambio, monto, fecha, email):
        super().__init__(monto, fecha, email)
        self.cambio = cambio
        self.monto = monto
        self.fecha = fecha
        self.email = email

